src/fpm/graph.py

```diff
@@ -5,9 +5,20 @@
 
 import rdflib
 from rdflib import RDF
+from rdflib.tools.rdf2dot import rdf2dot
 
 from fpm import traversal
-from fpm.constants import GEO, GEOM, COORD, COORD_EXT, QUDT, QUDT_VOCAB, FP
+from fpm.constants import (
+    GEO,
```

Comment:
I believe some of these namespaces are defined in [`rdf-utils.namespace`](https://github.com/minhnh/rdf-utils/blob/main/src/rdf_utils/namespace.py), if you want to reuse them. I do plan to migrate some more geometry model parsing there as well, but if you plan to further clean up later, I don't think it's too critical to leave these as it is for now. This PR, in any case, has a lot of changes to other parts of the package.

Reply:
I've opened #13 to track this separately